master/internal/api_command.go

```diff
@@ -132,6 +132,14 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 	var userFiles archive.Archive
 	if len(req.Files) > 0 {
 		userFiles = filesToArchive(req.Files)
+
+		workdirSetInReq := config.Environment.WorkDir != nil &&
```

Comment:
sorry, I'm a bit confused, the case where `config.Environment.WorkDir != nil && *workDirInDefaults == *config.Environment.WorkDir` is ok?

Reply:
i guess my question is, why is this different than the check for experiments? sorry i'm out of the loop product side on this one.

Reply:
i can't read